src/cpp/addressatcommand.cpp

```diff
@@ -41,8 +41,10 @@ CommandReply AddressAtCommand::run_me(RedisServer* server)
 Command* AddressAtCommand::clone()
 {
     AddressAtCommand* new_cmd = new AddressAtCommand(*this);
-    if (new_cmd == NULL){
+    if (new_cmd == NULL) {
         throw std::bad_alloc();
     }
     return new_cmd;
 }
+
+// EOF
```

Comment:
I might skip the ``// EOF `` comment since we aren't doing it in other files.

Reply:
see above